addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2125,218 +2125,214 @@ we-select.o_we_border_preview_aligned_select {
-                    height: $o-we-handle-edge-size;
+                    inset: $o-we-handles-offset-to-hide 0 auto auto;
+                    transform: translate(50%, -50%);
                     cursor: nesw-resize;
                 }
+
                 &.se {
-                    @include o-position-absolute(auto, $o-we-handle-border-width, -$o-we-handles-offset-to-hide - $o-we-handle-border-width, auto);
-                    height: $o-we-handle-edge-size;
+                    inset: auto 0 $o-we-handles-offset-to-hide * -1 auto;
+                    transform: translate(50%, 50%);
                     cursor: nwse-resize;
-
-                    &:after {
-                        @include o-position-absolute($top: 50%, $right: 40%);
-                    }
                 }
+
                 &.sw {
-                    @include o-position-absolute(auto, auto, -$o-we-handles-offset-to-hide - $o-we-handle-border-width/2, $o-we-handle-border-width);
-                    height: $o-we-handle-edge-size;
+                    inset: auto auto $o-we-handles-offset-to-hide * -1 0;
+                    transform: translate(-50%, 50%);
                     cursor: nesw-resize;
-
-                    &:after {
-                        @include o-position-absolute($left: 40%, $bottom: 40%);
-                    }
                 }
+
                 &.nw {
-                    @include o-position-absolute($o-we-handles-offset-to-hide - $o-we-handle-border-width, auto, auto, $o-we-handle-border-width + 1);
-                    height: $o-we-handle-edge-size;
+                    inset: $o-we-handles-offset-to-hide auto auto 0;
+                    transform: translate(-50%, -50%);
                     cursor: nwse-resize;
-
-                    &:after {
-                        @include o-position-absolute($left: 50%, $bottom: 40%);
-                    }
                 }
 
-                &.ne,
-                &.sw {
-                    content: "\f065";
-                }
-
-                &.nw,
-                &.se {
-                    content: "\f065";
-                    transform: rotate(90deg);
-                }
-
-                &::after {
-                    z-index: 1;
+                .o_handle_indicator {
+                    position: absolute;
+                    inset: $o-we-handles-btn-size * -0.5;
                     display: block;
-                    width: $o-we-handles-btn-size;
                     height: $o-we-handles-btn-size;
-                    border: solid 1px darken($o-we-handles-accent-color, 20%);
-                    line-height: $o-we-handles-btn-size - 2;
-                    font-size: 14px;
-                    font-family: FontAwesome;
-                    background-color: darken($o-we-handles-accent-color, 10%);
-                    color: white;
-                }
+                    width: $o-we-handles-btn-size;
```

Comment:
Nitpicking: I prefer `width` before `height` in general (consistency with the rest, common practice) but in this case, it would also allow to not break the history of that line for no reason :wink:

Reply:
Done 👍